packages/venia-concept/src/components/Checkout/paymentsFormItems.js

```diff
@@ -79,6 +81,7 @@ const PaymentsFormItems = props => {
                     />
                 </Field>
             </div>
+            <span ref={anchorRef} />
```

Comment:
I like this. I just wanted to point out that we are expanding this to be 5 lines of form inputs in #1286. I checked and this code will function fine, even if the form is 6 lines long. Beyond that, however, the "Billing Information" banner hides the top of the form. If you could make this resilient to that, such as if the top field/label were scrolled just under the header banner, then that would be best, however I don't think it's worth holding up the PR since, as I said, it'll work as is (3 lines of form inputs) and will work even up to 6 lines.